content/docs/command-reference/status.md

```diff
@@ -1,9 +1,8 @@
 # status
 
-Show changes in the <abbr>project</abbr>
-[pipelines](/doc/command-reference/dag), as well as file mismatches either
-between the <abbr>cache</abbr> and <abbr>workspace</abbr>, or between the cache
-and remote storage.
+Show changes in the <abbr>project</abbr> [pipelines](/doc/user-guide/pipelines),
```

Comment:
Minor: Actually this link was intentional. I try to keep the cmd ref self-contained not just within each page but as a whole i.e. only link to guides or other parts of docs in specific places like "...to learn more" admonitions. This is not necessarily an established or consistent practice so far though, but WDYT? We can add it to the ref. best practices.

Reply:
Oh, did not realize that and just violated the same principle in #3971. Not sure I have a strong opinion about it generally. What is the rationale for keeping the cmd ref self-contained?

In this case, I don't really find it linking to `dag` to be helpful, so I'd like to keep the link to the guide even if in general we try to keep cmd ref self-contained.

Reply:
> I don't really find it linking to dag to be helpful, so I'd like to keep the link to the guide

🆗 

Note that there's also a tootlip i.e. `<abbr>pipelines</abbr>` but the previous term (`project`) is already tooltip-ed and we probably don't want both tooltips together. So either keep as-is or tooltip `pipelines` only.

Again, this is minor 😬